bentoml/__init__.py

```diff
@@ -100,6 +117,7 @@
     "Context",
     "Service",
     "models",
+    "io",
```

Comment:
This is for using io descriptors directly from `bentoml` module, without the `from bentoml.io import *` line. E.g. `@svc.api(input=bentoml.io.NumpyNdarray(), ..)`

Used this in the example here: https://github.com/bentoml/gallery/pull/134/files#diff-fb9ea5d7763dd2f3544025fb18c4b89f335597a4cefa78f96c8eb849cb3f76ceR1

Note that IO descriptors implementation already does lazy load for heavy and optional dependencies, and doesn't increase the cost of loading bentoml module since `from bentoml.io import XXX` is identical to this.